src/store/collections/uploader/collection-uploader-actions.ts

```diff
@@ -21,8 +27,40 @@ export const collectionUploaderActions = unionize({
     SET_UPLOAD_PROGRESS: ofType<{ fileId: number, loaded: number, total: number, currentTime: number }>(),
     CLEAR_UPLOAD: ofType()
 }, {
-    tag: 'type',
-    value: 'payload'
-});
+        tag: 'type',
+        value: 'payload'
+    });

```

Comment:
Could you change sth in your VSC to not reformat this ? :)

Reply:
@unodgs Can we wrap unionize so that we wouldn't have to use tag and value config at all?